### DISCUSSION TITLE
Contribution activity's year does not show completely

### DISCUSSION THREAD

**Discussion by ZhiyuanMa2017** [community/community#33102] [Other Feature Feedback, Questions, & Ideas]
When you jump to one user's main page by clicking this user's name, the year will not show completely. 

For example, if you click "Andreas Jansson" from https://github.com/goodfeli?tab=following, the page will be like:

<br>
<img width="794" alt="image" src="https://user-images.githubusercontent.com/29819347/190295020-52f68a3f-4521-42f8-982f-f7922217cac1.png">

**Comment by radar-blip** [comment]
Its a bug and has already been reported a few times.
If you would like to file a bug report yourself, you can do so here: [Bug-Report](https://support.github.com/contact/bug-report)

Mine is still like this, has been for a couple weeks.
![buggy](https://user-images.githubusercontent.com/39196767/190310656-67705565-2451-461b-a7fc-66c2200deb72.png)

**Comment by jfullstackdev** [comment]
I think this is already fixed, what is your browser ? I'm using Chrome, I just experienced that some time ago

**Reply by ZhiyuanMa2017** [reply]
Not fixed for me. I reported this bug to GitHub as linuxlawson said, and the GitHub Support replied that the engineers are looking into this.

**Comment by ZhiyuanMa2017** [comment]
Fixed now.

**Reply by radar-blip** [reply]
Mine is back to normal too.
